src/client/gui/lib/copyable_text.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart' hide Tooltip;
+import 'package:flutter/services.dart';
+import 'tooltip.dart';
```

Comment:
please separate the imports that come from external libraries from the ones that come from our own code